### PR TITLE
Return of the old docs banner

### LINKED ISSUE #25244: Old-docs banner not working in 2.5.2 and newer
Likely because we missed out on porting it to the new paradox docs.

Working: https://doc.akka.io/docs/akka/2.5.1/
missing: https://doc.akka.io/docs/akka/2.5.2/

### PR DESCRIPTION
**Pull request by johanandren**
Fixes #25244 

Some screenshots:

screen shot 2018-06-19 at 21 30 21
screen shot 2018-06-19 at 21 36 00
screen shot 2018-06-19 at 22 04 09

### CODE REVIEW

**Comment by johanandren** [comment]
A bit tricky to test since akka.io doesn't allow cross site requests for `versions.json`, but I've been running against a local Akka HTTP mimicking the docs server paths with a local copy of `versions.json` to verify

**Comment by 2m** [comment]
We have added such a warning to the generic theme: https://github.com/lightbend/paradox/pull/210 where it does not require to keep a separate version file.

**Comment by johanandren** [comment]
Given that we already have the infra in place for the version file I'm not sure we gain anything by not keeping using it?

**Comment by 2m** [comment]
Yea, I agree. Bringing it back makes most sense.

**Review by ktoso — APPROVED** [review]
LGTM, awesome come back of the warnOldDocs.js 😉

**Review by raboof — APPROVED** [review]
:+1: nice.

I've always found it confusing that there are 2 links in "Please upgrade to version 2.5.13 as soon as possible.

Click here to go to the same page on the 2.5.13 version of the docs.", one going to the index and one going to the 'same page' - I think we should perhaps only link to the 'same page', but that could be a separate PR.

File: akka-docs/src/main/paradox/assets/js/warnOldDocs.js [hunk 1]
```diff
@@ -0,0 +1,186 @@
+jQuery(document).ready(function ($) {
+
+  function initOldVersionWarnings($) {
+    $.get("//akka.io/versions.json", function (akkaVersionsData) {
```
Comment (raboof) [inline akka-docs/src/main/paradox/assets/js/warnOldDocs.js]: There's no error reporting when this fails or is disallowed by the browser - that seems OK though, as we gracefully handle it by not showing a banner

**Review by raboof — APPROVED** [review]

**Pull request merged**